account_payment_return_financial_risk/readme/USAGE.md

```diff
@@ -0,0 +1,10 @@
+To use this module, you need to:
+
+1.  Go to *Customers \> Financial Risk*
+2.  Set limits and choose options to compute in credit limit.
+3.  Create an invoice and pay it.
+4.  Create a payment return.
+5.  Go to *Invoicing/Accounting \> Customers \> Invoices* and create new
+    customer invoices.
+6.  Test the restriction trying to create an invoice for the partner for
+    an amount higher of the limit you have set.
```

Comment:
It is missing the steps you need to set up which field in financial risk.
Maybe is needed a CONFIGURATION.md, to explain we have to add groups to user Financial/Manager to allow to edit the amounts, and which field we need to set up.

Also here is missing the step to create a customer return

Reply:
It is in the original module, so I don't think we need to add the same in this module if it is only an extension.

https://github.com/OCA/credit-control/blob/17.0/account_financial_risk/readme/CONFIGURE.md

Reply:
You can add a mention saying "Refer to the module ... configuration for initial steps.".